README.md

```diff
@@ -2,13 +2,13 @@
 
 <br />
 
-PostHog provides open source product analytics, built for developers. Automate the collection of every event on your website or app, with no need to send data to 3rd parties. It's just 1 click to deploy on your own infrastructure, with full API/SQL access to the underlying data. 
- 
-<br /><br /> 
- 
+PostHog provides open-source product analytics, built for developers. Automate the collection of every event on your website or app, with no need to send data to 3rd parties. Just 1 click to deploy on your own infrastructure, with full API/SQL access to the underlying data. 
```

Comment:
Starting the sentence with "Just" here sounds a bit off for me. If you're not a fan of "It's just", perhaps we should reword to use "With just"